### PR TITLE
Topic sk bug 35241678

### PR DESCRIPTION
**Pull request by skommala**
Fixed Bug 35241678 - Network validation script flagging load balancer port 443 port to be open to Internet.

1. Tested private load balancer with out cidr 0.0.0.0/0
2. Tested public load balancer.

### CODE REVIEW

File: terraform/modules/network-validator/scripts/network_validation.sh [hunk 1]
```diff
@@ -519,9 +523,10 @@ This script is used to validate existing subnets, and optionally network securit
   -g, --lpg           OCID of the Local Peering Gateway (LPG) in the DB VCN
   -b, --bastionsubnet Bastion Subnet OCID
   -i, --bastionip     Bastion Host IP. Provide this if using existing bastion
+  -j, --lbdestinationcidr   Indicates use of private load balancer(true or false)
```
Comment (roberto-sanchez-herrera) [inline terraform/modules/network-validator/scripts/network_validation.sh:526]: this description does not look correct.

File: terraform/main.tf [hunk 2]
```diff
@@ -21,6 +21,7 @@ module "network-validation" {
   existing_lb_nsg_id             = var.add_existing_nsg && var.add_load_balancer ? var.existing_lb_nsg_id : ""
   existing_mount_target_nsg_id   = var.add_existing_nsg && var.add_fss ? var.existing_mount_target_nsg_id : ""
   existing_bastion_nsg_id        = var.add_existing_nsg && var.is_bastion_instance_required ? var.existing_bastion_nsg_id : ""
+  lb_destination_cidr            = var.add_load_balancer ? (var.is_lb_private ? "" : "0.0.0.0/0") : ""
```
[the remark this answers is not in the document]
Comment (abhijit-paranjpe) [reply terraform/main.tf:24]: I am finding it confusing. What name are we using else where?

**Comment by abhijit-paranjpe** [comment]
Looks good for the most part. Just the naming of the variable if you could address it.

File: terraform/modules/network-validator/scripts/network_validation.sh (hunk 1 above)
Comment (abhijit-paranjpe) [reply terraform/modules/network-validator/scripts/network_validation.sh]: @skommala Is this still pending?

Comment (skommala) [reply terraform/modules/network-validator/scripts/network_validation.sh]: Fixed it.

File: terraform/main.tf (hunk 2 above)
[the remark this answers is not in the document]
Comment (skommala) [reply terraform/main.tf]: Fixe dit.

**Comment by skommala** [comment]
@abhijit-paranjpe @roberto-sanchez-herrera Addressed the comments.

**Review by roberto-sanchez-herrera — APPROVED** [review]

**Pull request merged**